frontend/src/metabase/dashboard/hooks/use-location-sync.ts

```diff
@@ -4,36 +4,36 @@ import { replace } from "react-router-redux";
 import { usePrevious } from "react-use";
 import { omit } from "underscore";
 
-import { DEFAULT_EMBED_DISPLAY_OPTIONS } from "metabase/dashboard/constants";
+import { DEFAULT_DASHBOARD_DISPLAY_OPTIONS } from "metabase/dashboard/constants";
 import { parseHashOptions, stringifyHashOptions } from "metabase/lib/browser";
 import { useDispatch } from "metabase/lib/redux";
 import { isNullOrUndefined } from "metabase/lib/types";
 
 type SYNCED_KEY = "refresh" | "fullscreen" | "theme";
 
-const DEFAULT_DASHBOARD_EMBED_DISPLAY_OPTIONS = {
-  theme: DEFAULT_EMBED_DISPLAY_OPTIONS.theme,
+const DEFAULT_SYNCED_DASHBOARD_OPTIONS = {
+  theme: DEFAULT_DASHBOARD_DISPLAY_OPTIONS.theme,
```

Comment:
Previously, `DEFAULT_EMBED_DISPLAY_OPTIONS` was used here, however, we could notice that this file has nothing to do with just _embedding_` this file is used in an actual dashboard as well, so the constant name should be changed to prevent confusion that it might have been only used in embedded dashboard.